tests/ha/multi_pod_failure/test_multi_data_server_pod_failure.py

```diff
@@ -0,0 +1,635 @@
+                            " node", node_name, HA_CFG["common_params"]["pod_joinback_time"])
+                time.sleep(HA_CFG["common_params"]["pod_joinback_time"])
+        if self.restore_ip:
+            for node_ip in self.node_ip_list:
+                LOGGER.info("Cleanup: Get the network interface up for %s ip", node_ip)
+                node_iface = self.ip_dict.get(node_ip)[0]
+                worker_obj = self.ip_dict.get(node_ip)[1]
+                worker_obj.execute_cmd(cmd=cmd.IP_LINK_CMD.format(node_iface, "up"),
+                                       read_lines=True)
+                resp = sysutils.check_ping(host=node_ip)
+                assert_utils.assert_true(resp, "Interface is still not up.")
+        if os.path.exists(self.test_dir_path):
+            sysutils.remove_dirs(self.test_dir_path)
+        # TODO: As cluster restart is not supported until F22A, Need to redeploy cluster after
+        #  every test
+        if self.deploy:
+            LOGGER.info("Cleanup: Destroying the cluster ")
+            resp = self.deploy_lc_obj.destroy_setup(self.node_master_list[0],
+                                                    self.node_worker_list,
+                                                    const.K8S_SCRIPTS_PATH)
+            assert_utils.assert_true(resp[0], resp[1])
+            LOGGER.info("Cleanup: Cluster destroyed successfully")
+
+            LOGGER.info("Cleanup: Setting prerequisite")
+            self.deploy_lc_obj.execute_prereq_cortx(self.node_master_list[0],
+                                                    const.K8S_SCRIPTS_PATH,
+                                                    const.K8S_PRE_DISK)
+            for node in self.node_worker_list:
+                self.deploy_lc_obj.execute_prereq_cortx(node, const.K8S_SCRIPTS_PATH,
+                                                        const.K8S_PRE_DISK)
+            LOGGER.info("Cleanup: Prerequisite set successfully")
+
+            LOGGER.info("Cleanup: Deploying the Cluster")
+            resp_cls = self.deploy_lc_obj.deploy_cluster(self.node_master_list[0],
+                                                         const.K8S_SCRIPTS_PATH)
+            assert_utils.assert_true(resp_cls[0], resp_cls[1])
+            LOGGER.info("Cleanup: Cluster deployment successfully")
+
+        LOGGER.info("Cleanup: Check cluster status")
+        resp = self.ha_obj.check_cluster_status(self.node_master_list[0])
+        assert_utils.assert_true(resp[0], resp[1])
+        LOGGER.info("Cleanup: Cluster status checked successfully")
+
+        LOGGER.info("Done: Teardown completed.")
+
+    # pylint: disable=too-many-locals
+    # pylint: disable=too-many-statements
+    @pytest.mark.ha
+    @pytest.mark.lc
+    @pytest.mark.tags("TEST-35792")
+    @pytest.mark.skip(reason="Buckets cruds won't be supported with DTM0")
+    def test_server_data_kpods_fail_during_ios(self):
```

Comment:
do we not have any test for "after" scenario?

Reply:
NO. This during is the only test where we are performing shutdown of both server and data pods.